AnnoyancesFilter/sections/cookies_specific.txt

```diff
@@ -5185,7 +5185,7 @@ ipaper.io#%#//scriptlet('remove-class', 'noScroll', 'html.noScroll')
 comidoc.net#%#//scriptlet('set-cookie', 'consent', 'true')
 joinhoney.com#%#//scriptlet('set-cookie', 'acceptedCookiePolicy', '1')
 pcworld.com#%#AG_onLoad(function() { setTimeout(function() { var el = document.querySelector('button[aria-label="I Agree"]'); if(el) { el.click(); } }, 1000); });
-rtl.de#%#AG_onLoad(function() { setTimeout(function() { var el = document.querySelector('button[title="Alle akzeptieren"]'); if(el) { el.click(); } }, 1000); });
+n-tv.de,rtl.de#%#AG_onLoad(function() { setTimeout(function() { var el = document.querySelector('button[title="Alle akzeptieren"]'); if(el) { el.click(); } }, 1000); });
```

Comment:
IMHO It's better to use the rule with MutationObserver to avoid blinking, for example this:
`#%#AG_onLoad(function() {var b=new MutationObserver(function() {var c=document.querySelector('button[title="Alle akzeptieren"]'); c&&(b.disconnect(),c.click()) });b.observe(document, {childList: !0,subtree: !0});setTimeout(function() {b.disconnect()}, 1E4)})`

Reply:
@Sergey-Lyapin @stanislav-atr have you checked it in Incognito mode with German IP? I have a broken scrool at first visit.
<details><summary>Screenshot:</summary>

![image](https://user-images.githubusercontent.com/33488229/108485907-7c498100-72ae-11eb-845c-3a8d4c80b322.png)

</details><br/>

Reply:
This rule should work fine
```
n-tv.de#%#AG_onLoad(function() {if (!document.cookie.includes("euconsent-v2")) {var g = new MutationObserver(function() {var b = document.querySelector('button[title="Alle akzeptieren"]');var c = document.querySelector('html');b && b.click();c && c.style.setProperty('overflow', 'visible', 'important');c && c.style.setProperty('position', 'static', 'important');});g.observe(document, {childList: !0,subtree: !0});setTimeout(function() {g.disconnect()}, 1E4)}});
```

Reply:
> @Sergey-Lyapin @stanislav-atr have you checked it in Incognito mode with German IP? I have a broken scroll at first visit.
> 
> Screenshot:

Yes, and scrolling worked and works good

Reply:
Weird, I still have it. @AdamWr could you please check it on your end?

Reply:
@zloyden, scroll works fine on my end.
<details><summary>Screenshot:</summary>

![image](https://user-images.githubusercontent.com/68852737/108490911-51622b80-72b4-11eb-8069-619fef1c077a.png)
</details><br/>

Reply:
This rule - https://github.com/AdguardTeam/AdguardFilters/pull/75218#discussion_r579048035 works fine on my end.
If it still doesn't work on your side then we can add `n-tv.de` to this rule:
https://github.com/AdguardTeam/AdguardFilters/blob/373ecb946bd8cb29f28450503ef600dfadec6d66/AnnoyancesFilter/sections/cookies_specific.txt#L85
like in other similar cases.

Reply:
> This rule - [#75218 (comment)](https://github.com/AdguardTeam/AdguardFilters/pull/75218#discussion_r579048035) works fine on my end.

 I have the same. @stanislav-atr you should exchange your rule to this https://github.com/AdguardTeam/AdguardFilters/pull/75218#discussion_r579047296.